esmvaltool/_recipe.py

```diff
@@ -402,10 +402,6 @@ def _update_fx_settings(settings, variable, config_user):
     for step in ('average_region', 'average_volume'):
         if step in settings and settings[step].get('fx_files') is not False:
```

Comment:
Change `False` to `None`, this should also fix the tests :+1:

Reply:
Amazing!